game/static/game/css/contributors.css

```
@@ -0,0 +1,573 @@
+  flex: 1;
+}
+
+.contributor-name {
+  font-size: 1.2rem;
+  font-weight: 700;
+
+  color: white;
+
+  margin-bottom: 0.5rem;
+
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)